### DISCUSSION TITLE
升级openresty,容器版本信息读取错误,导致一直提示升级.

### DISCUSSION THREAD

**Discussion by AtlantisDe** [1Panel-dev/1Panel#5627] [Q&A]
### 1Panel 版本

v1.10.11-lts

### [BUG] 升级openresty,容器版本信息读取错误,导致一直提示升级.

- 本来我不想提交为 Issue 的,但是我认为应该是BUG 所以你们需要检查下.
- 之前 这里 发布为讨论 https://github.com/1Panel-dev/1Panel/discussions/5616
- 但是我实际升级后,发现,容器已经使用了最新版的镜像.但是一直提示升级.我认为是读取配置错误.

<img width="525" alt="9f13cd685a85f39b159f7cac6d25ca8" src="https://github.com/1Panel-dev/1Panel/assets/39150678/0b70f98b-f1df-4df2-b80d-45695b8ffc48">

<img width="1253" alt="9132736d6a1131c0d8cf14f52202d77" src="https://github.com/1Panel-dev/1Panel/assets/39150678/92af06a7-07f3-4f7c-8a0b-6bdc48fafe94">

### 重现步骤

升级到最新版 1.21.4.3-3-3-focal

**Comment by wanghe-fit2cloud** [comment]
本地环境没能复现该问题。
> 可以尝试 `升级应用` 或者点击 `重建` 按钮，看是否能解决该问题。
